arch/arm/src/stm32/stm32_fdcan_sock.c

```diff
@@ -2128,7 +2127,7 @@ static void fdcan_error_work(void *arg)
 
   /* Re-enable ERROR interrupts */
 
-  fdcan_errint(priv, true);
+  fdcan_putreg(priv, STM32_FDCAN_IE_OFFSET, regval);
```

Comment:
@csanchezdll shouldn't it be "ie"? I didn't find "regval" variable declared and set here.